sql/_23_apricot_qa/_03_i18n/tr_TR/_09_identifiers/answers/_010_identifiers_show_003.answer

```diff
@@ -14,36 +14,36 @@
 0
 ===================================================
```

Comment:
Shouldn't it be changed to the same content below?
```
sql/_23_apricot_qa/_03_i18n/tr_TR/_09_identifiers/answers/_010_identifiers_show_003.answer_cci
```

Reply:
Reflected the review. Please check again.